sim/mcuboot-sys/src/api.rs

```diff
@@ -1,104 +1,198 @@
 //! HAL api for MyNewt applications
 
 use simflash::{Result, Flash, FlashPtr};
-use lazy_static::lazy_static;
```

Comment:
This does remove all uses of lazy_static, so it is good to remove that from the Cargo.toml file as well.

Reply:
done